### ISSUE TITLE
v0.9.1 "export name annotations" generates js output with syntax error

### ISSUE THREAD

**Issue by Shigma** [evanw/esbuild#960]
### Input

```ts
export default xxx
export const name = xxx
export function apply() {}
```

### Output

```js
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  apply,
  default, // syntax error here
  name
});
```

**Comment by vladmandic** [comment]
even simpler here, generates syntax error:
```js
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  default
});
```

**Comment by evanw** [comment]
Crap, sorry about that. I will fix this very soon.

**Comment by evanw** [comment]
This has been fixed in version 0.9.2.
